### PR TITLE
fix: longer refresh intervals

### PR DESCRIPTION
**Pull request by pauldambra**
## Problem

Auto refresh of dashboards is too aggressive. See: https://posthog.slack.com/archives/C0368RPHLQH/p1675846412082249

We agreed to make the auto refresh intervals longer.

## Changes

* Makes the auto refresh interval options: 30m, 40m , 50m, 60m
* Removes a bunch of ANTD from the dropdown (without getting rabbitholed into replacing Radio.Group)
* changes from checkbox to a switch

Screenshot 2023-02-08 at 10 53 06
Screenshot 2023-02-08 at 10 52 54

## How did you test this code?

👀

### CODE REVIEW

**Comment by pauldambra** [comment]
@macobo it's a messy diff but the two commits are pretty tidy in comparison. 1st to change functionality, 2nd to squash some ants

File: frontend/src/scenes/dashboard/DashboardReloadAction.tsx [hunk 1]
```diff
@@ -30,74 +31,76 @@ const intervalOptions = [
 export function DashboardReloadAction(): JSX.Element {
     const { itemsLoading, autoRefresh, refreshMetrics } = useValues(dashboardLogic)
```
Comment (macobo) [inline frontend/src/scenes/dashboard/DashboardReloadAction.tsx:32]: We probably should rename `autoRefresh` to bust localStorage.

**Review by macobo — APPROVED** [review]
:clap:

**Review by yakkomajuri — APPROVED** [review]
Very nice! I'd personally go for larger steps. i.e. 30min, 45min, 1h, 3h

A key question here though is: how do we incentivize users to pick any value other than the lowest possible? The other values only benefit us, not the customer, so there's an inherent conflict here

**Comment by pauldambra** [comment]
Yeah, I wondered about longer steps but I can't think of why I'd want a 3h refresh over a 1h refresh...

I think you're spot on in #14143 that there's a conflict between freshest data and lowest load (at the moment at least)

**Pull request merged**